install/installation-archlinux.md

```diff
@@ -0,0 +1,130 @@
+# Install self-hosted TimescaleDB on archlinux-based systems
+You can host TimescaleDB yourself, on your Arch Linux system. These
+instructions use the `pacman` package manager.
+
+<highlight type="warning">
+If you have already installed PostgreSQL using a method other than the `pacman`
+package manager, you could encounter errors following these instructions. It is
+safest to remove any existing PostgreSQL installations before you begin. If you
+want to keep your current PostgreSQL installation, do not install TimescaleDB
+using this method.
+[Install from source](/install/latest/self-hosted/installation-source/)
+instead.
+</highlight>
+
+<procedure>
+
+### Installing self-hosted TimescaleDB on archlinux-based systems
+1.  At the command prompt, as root, add the PostgreSQL third party repository
+    to get the latest PostgreSQL packages:
+    ```bash
+    pacman -Syu timescaledb
+    ```
+1.  Update the `postgresql.conf` file, to use TimescaleDB as PostgreSQL extension:
+    ```bash
+    echo "shared_preload_libraries = 'timescaledb'" >> /var/lib/postgres/data/postgresql.conf
+    ```
+1.  Install `timescaledb-tune`:
+    ```bash
+    pacman -Syu timescaledb-tune
+    ```
+1.  As the `postgres` user, connect to the PostgreSQL instance:
+    ```bash
+    su -l postgres
+    ```
```

Comment:
This command does not connect to the postgres instance but switches to the postgres user